source/common/network/udp_listener_impl.h

```diff
@@ -39,6 +47,7 @@ class UdpListenerImpl : public BaseListenerImpl {
 
 private:
   void onSocketEvent(short flags);
+  void writev(const Buffer::RawSlice* slices, uint64_t num_slice);
```

Comment:
Can we return the SysCallSizeResult? In send() if an write error happen, we might want to call callbacks.onError()

Reply:
sure